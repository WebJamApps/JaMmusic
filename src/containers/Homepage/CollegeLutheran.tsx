import React from 'react';

const CollegeLutheran = (): JSX.Element => (
  <div className="elevation2 project" style={{ minWidth: '313px' }}>
    <h4>College Lutheran Church</h4>
    <img
      alt="College Lutheran Church Homepage"
      src="https://dl.dropboxusercontent.com/s/354maifx0dkzt9s/Screenshot%20from%202019-02-22%2016-01-50.png?dl=0"
    />
    <br />
    <a
      href="https://www.collegelutheran.org"
      target="_blank"
      rel="noreferrer noopener"
    >
      Collegelutheran.org
    </a>
  </div>
);

export default CollegeLutheran;
