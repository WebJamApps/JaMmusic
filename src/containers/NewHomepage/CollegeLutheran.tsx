function CollegeLutheran(): JSX.Element {
  return (
    <div
      style={{ boxShadow: '0 4px 5px 0 rgba(0, 0, 0, 0.14), 0 1px 10px 0 rgba(0, 0, 0, 0.12), 0 2px 4px -1px rgba(0, 0, 0, 0.4)', height: '500px' }}
    >
      <img
        style={{ width: '95%', height: '455px' }}
        alt="College Lutheran Church Homepage"
        src="https://dl.dropboxusercontent.com/s/354maifx0dkzt9s/Screenshot%20from%202019-02-22%2016-01-50.png?dl=0"
      />
      <br />
      <a
        href="https://www.collegelutheran.org"
        target="_blank"
        rel="noreferrer noopener"
      >
        Collegelutheran.org
      </a>
    </div>
  );
}

export default CollegeLutheran;
