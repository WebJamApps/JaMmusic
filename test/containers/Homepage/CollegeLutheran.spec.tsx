
import { shallow } from 'enzyme';
import CollegeLutheran from '../../../src/containers/Homepage/CollegeLutheran';

const wrapper = shallow<typeof CollegeLutheran>(<CollegeLutheran />);

describe('CollegeLutheran', () => {
  it('Renders CollegeLutheran', () => {
    expect(wrapper.find('div.elevation2.project').exists()).toBe(true);
    expect(wrapper).toMatchSnapshot();
  });
});
