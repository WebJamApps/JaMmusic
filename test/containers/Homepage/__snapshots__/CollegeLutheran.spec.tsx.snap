// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CollegeLutheran Renders CollegeLutheran 1`] = `ShallowWrapper {}`;
